R/geom_text_lastonly.R

```diff
@@ -128,13 +128,12 @@ geom_text_lastonly <- function(mapping = NULL, data = NULL,
 #' The \code{cmapplot} package contains a few custom ggproto objects. For the
 #' most part, these are slightly tweaked versions of ggplot2's default proto
 #' objects. For more information about these, see
-#' \code{\link[ggplot2]{ggplot2-ggproto}}.
+#' \code{\link[ggplot2::ggproto]{ggplot2::ggproto}}.

```

Comment:
@nmpeterson this change is now causing a warning for me in `check()`:
```
Missing link or links in documentation object 'customproto.Rd':
    '[ggplot2::ggproto]{ggplot2::ggproto}'
```

What links do and don't work is totally beyond me (I haven't done any testing of links and when I load help files in R the links haven't been active) but I think this help page is titled ggplot2-ggproto?

![image](https://user-images.githubusercontent.com/54633946/99439684-22b26e80-28db-11eb-8c14-990b20aab78e.png)

Reply:
Ah, crap, I must have forgotten the `2` when I was trying to bring up the help, and concluded that you intended to link to `?ggroto` (which also exists, but is for creating new ggproto objects). I will revert this.

Reply:
Should be fixed by commit 2246b3e3d1e72de56bd9f2c849db65de36c5dbae